docs/modules/candid-guide/index.adoc

```diff
@@ -289,23 +320,56 @@ If you want to create a new Candid implementation, this document will not suffic
 
 Additionally, the official https://github.com/dfinity/candid/tree/master/test[Candid test data] is useful to make sure your implementation is compatible with Candid, even in slightly more obscure corner cases.
 
-[#reference]
 == Reference
 
-This section lists all the types supported by Candid, including their type syntax, the syntax for the textual representation, their upgrading rules and the corresponding types in Rust, Motoko and Javascript.
+This section provides links to Candid tools and libraries for currently supported host languages and detailed reference information about Candid supported types.
```

Comment:
Hm, it feels odd to lump the description of tools under the Candid language reference section. I'd move it somewhere else.